app/javascript/packages/document-capture/hooks/use-form-change-completion.ts

```diff
@@ -0,0 +1,17 @@
+import { useContext } from 'react';
+import { FormStepsContext } from '@18f/identity-form-steps';
+
+function useFormChangeCompletion({
+  isSelfieCaptureEnabled,
+  docAuthSeparatePagesEnabled,
+}: {
+  isSelfieCaptureEnabled: boolean;
+  docAuthSeparatePagesEnabled: boolean;
+}) {
+  const { changeStepCanComplete } = useContext(FormStepsContext);
+  if (isSelfieCaptureEnabled && docAuthSeparatePagesEnabled) {
+    changeStepCanComplete(false);
```

Comment:
This doesn't really address the previous comment about side effects in the render path, because this function (and the resulting `changeStepCanComplete`) is still called as a direct result of render, each time the render happens.

You'll probably want [`useEffect`](https://react.dev/reference/react/useEffect) somewhere. We may not even need this separate hook if you'd rather just inline the `useEffect`.